carta/cpp/core/ScriptedClient/ScriptFacade.cpp

```diff
@@ -742,15 +742,19 @@ QStringList ScriptFacade::centerImage( const QString& controlId ) {
 }
 
 QStringList ScriptFacade::getCenterPixel( const QString& controlId ) {
-    QStringList resultList("");
+    QStringList resultList;
     Carta::State::CartaObject* obj = _getObject( controlId );
     if ( obj != nullptr ){
         Carta::Data::Controller* controller = dynamic_cast<Carta::Data::Controller*>(obj);
         if ( controller != nullptr ){
-            resultList = controller->getCenterPixel();
-            if ( resultList[0] == "null" ) {
+            QPointF center = controller->getCenterPixel();
+            if ( center.x() == -0.0 && center.y() == -0.0 ) {
```

Comment:
Negative 0 to indicate error condition? I personally would prefer nan. Anyone else have a comment on this?